aiidalab_widgets_base/process.py

```diff
@@ -302,7 +302,7 @@ def follow(self, detach=False):
 
         if self._monitor is None:
             self._monitor = ProcessMonitor(
-                process=self.process,
+                value=self.process.uuid,
```

Comment:
ah, my mistake, I did not check the class constructor. I removed this line. 

> I think the proper way is to make the process_uuid an argument of __init__ and remove the ipw.dlink below.

I don't think we can remove the `dlink`, it is needed when the `process` traitlet of ProcessFollowerWidget changes.

I do agree that we should add a possibility of passing the UUID through the constructor (but since the traitlet is called `value` it should be named value. This would make this consistent with how ipywidgets are designed. But note that almost none of the widgets in this file are done this way so this is for a longer discussion.

Reply:
I just realized that in the end this was not a bug, since this wrong constructor parameter was simply ignored as an extra keyword argument (and because we have the `dlink` below). That's why this was not a runtime error but just a warning in pytest. So should be safe to merge.